### ISSUE TITLE
Custom Field Value persistence seems to be broken for recent WSDL versions?

### ISSUE THREAD

**Issue by adambedford** [NetSweet/netsuite#207]
Linked to #196. 

I was not having success saving custom field values using the 2014_2 WSDL, but reverting to the 2012_2 version fixed this. Unfortunately, the 2012_2 schema is missing some recent additions, such as TimeSheet at TimeEntry. Sticking with the 2012_2 schema isn't a long-term solution so I'd like to see what it will take to get this library working with 2014_2 and custom field values saving.

Is this an issue others have experienced? I'm still not all that familiar with the NetSuite API nor SOAP APIs in general, but I'd definitely like to contribute if this is an issue that extends beyond my use.

**Comment by jcroteau** [comment]
Hi @adambedford , I also ran into this issue recently on the 2014_2 WSDL.  Adding/updating custom field values used to work just fine, but something changed after we recently refreshed our NS environments.

The problem for me seems to be that the original human-readable```internalId``` values on custom records were moved to ```scriptId```s, and the ```internalId``` took on (I assume) a system-generated numeric id. Looking at the NS response in your comment, [here](https://github.com/NetSweet/netsuite/issues/196#issuecomment-119271932), it appears it is the same for you.

Currently, the netsuite gem is setting the ```internalId```:
https://github.com/NetSweet/netsuite/blob/7801251c21379ddb9332f205f8aee7c30f3f1b5a/lib/netsuite/records/custom_field_list.rb#L149

It will have to set ```scriptId``` instead, though I'm not sure yet if that is sufficient or if the new numeric ```internalId``` will also need to be set, which would require prior knowledge of those id's :cry:
